include/openPMD/RecordComponent.hpp

```diff
@@ -137,9 +156,38 @@ class RecordComponent : public BaseRecordComponent
         Extent,
         double targetUnitSI = std::numeric_limits< double >::quiet_NaN() );
 
+    /** Store a chunk of data
+     *
+     * shared_ptr< T > for data must be contiguous and large enough for extent
+     *
+     * In FlushType::DEFER, the data managed by the passed
+     * std::shared_ptr< T > MUST NOT be modified or deleted until
+     * Series::flush is called.
+     *
+     * @todo implement: if the shared pointers' `use_count` drops to one
+     *       before Series::flush is called, the store will be skipped during
+     *       Series::flush.
```

Comment:
I don't think this works, though.
The user schedules a chunk store and then drops *his* ref to the data. So the write would still be valid.

Reply:
Ah true for writes!